.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,37 @@
-**IMPORTANT**: PRs without the template will be CLOSED
-
-### Description of your *pull request* and other information
-
 <!--
+    **IMPORTANT**: PRs without the template will be CLOSED
+    
+    Due to the high number of pull requests, it may take some time before your code will be reviewed.
+    It is recommended for new contributors to stick to smaller pull requests, to familiarize themselves with the codebase.
+    Large PRs take longer to be reviewed and merged.
```

Comment:
maybe something like this?
```suggestion
    Due to the high volume of pull requests, it may be a while before your PR is reviewed.
    Please try to keep your pull request focused on a single bugfix or new feature.
    Pull requests with a vast scope and/or very large diff will take much longer to review.
    It is recommended for new contributors to stick to smaller pull requests, so you can receive much more immediate feedback as you familiarize yourself with the codebase.
```